helm/botkube/values.yaml

```diff
@@ -504,6 +504,14 @@ executors:
       #      resources: [ "deployments", "pods", "namespaces", "daemonsets", "statefulsets", "storageclasses", "nodes", "configmaps", "services", "ingresses", "replicasets", "secrets", "cronjobs", "jobs" ]
       context: *default-plugin-context
 
+  bins-management:
```

Comment:
What do you think about adding some comments to newly added fields? So that, it will add proper docs to chart readme.